web/handlers/http.go

```diff
@@ -255,16 +247,23 @@ func New(
 
 	mainMux.Handle("/", m)
 
-	// apply middleware
-	var finalHandler http.Handler = mainMux
-	finalHandler = logging.InjectHandler(logger)(finalHandler)
-	finalHandler = CSRF(finalHandler)
+	// apply HTTP middleware
+	middlewares := []func(http.Handler) http.Handler{
+		logging.InjectHandler(logger),
+		user.ContextInjecter(fs, a),
+		CSRF,
+	}
+
+	if !web.Production {
+		middlewares = append(middlewares, r.GetReloader())
+	}
 
-	if web.Production {
-		return finalHandler, nil
+	var finalHandler http.Handler = mainMux
+	for _, mw := range middlewares {
+		finalHandler = mw(finalHandler)
```

Comment:
```suggestion
	for _, applyMiddleware := range middlewares {
		finalHandler = applyMiddleware (finalHandler)
```
(lmk if i misunderstood!)